components/eventing-controller/Makefile

```diff
@@ -0,0 +1,125 @@
+APP_NAME = eventing-controller
+APP_PATH = components/$(APP_NAME)
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-kubebuilder2:v20200831-e46c648b
+SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/common/makefiles
+
+# DEV_IMG is the image used during development workflow only
+DEV_IMG = ko://github.com/kyma-project/kyma/components/eventing-controller
+override ENTRYPOINT = cmd/main.go
```

Comment:
I think `cmd/main.go` should be `main.go`.

Reply:
Fixed

Reply:
Fixed